devlib/trace/perf.py

```diff
@@ -104,7 +104,11 @@ def start(self):
             self.target.kick_off(command)
 
     def stop(self):
+        self.target.killall('perf', signal='SIGINT',
+                            as_root=self.target.is_rooted)
+        # perf doesn't transmit the signal to its sleep call so handled here:
         self.target.killall('sleep', as_root=self.target.is_rooted)
```

Comment:
`signal='SIGINT'` could also be used here but we're allowed to be more violent with `sleep`!